web/src/components/ExpandedDetail.js

```diff
@@ -26,100 +31,178 @@ const theme = createMuiTheme({
   }
 })
 
+const linkTheme = createMuiTheme({
+  palette: {
+    primary: {
+      main: '#000000'
+    },
+    secondary: {
+      main: '#FEAF0A'
+    }
+  }
+})
+
+const nestedStyle = {
+  paddingLeft: theme.spacing(4)
+}
+
 const imgAlt = 'not found'
 
 class ExpandedDetail extends React.Component {
+
   constructor(props) {
     super(props)
-    this.state = {}
+    this.state = {
+      podsVisible: false,
+      resourcesVisible: false,
+      pods: null,
+      resources: null
+    }
   }
 
-  render() {
-    return (
-      <MuiThemeProvider theme={theme} >
-        <div className="flex-container">
-          <div className="helm-status">
-            <div className="right-padded">
-              <img src={HelmIcon} width="32" height="32" alt={imgAlt} />
-            </div>
-            <div>
-              <Typography variant="h5">ChartVersion</Typography>
-            </div>
-          </div>
+  podClick = () => {
+    this.setState({podsVisible: !this.state.podsVisible})
+    let API_ADDRESS = urljoin('https://' + window.location.host, '/releases/', this.props.data.name, 'resources') // point to local IP for testing
```

Comment:
It's smelly that we build the API address in 2 places. It'd be cleaner if the root `https://${window.location.host}/api` was only defined once, and then the `ExpandedDetail` & `App` components could append their specific routes on top of it.

Is the `API_ADDRESS` const from `App.js` in scope here? If so, we could have that const be only the root, instead of the root + `/releases` endpoint, and share it between the components.

Reply:
The API address is already fed into the Grid component as a prop. I will make use of this and feed it from the grid to the detail when it is rendered.